static/bidder-info/lemmadigital.yaml

```diff
@@ -1,4 +1,12 @@
-endpoint: "https://sg.ads.lemmatechnologies.com/lemma/servad?pid={{.PublisherID}}&aid={{.AdUnit}}"
+# We have the following regional endpoint domains: 
+# "uses" (us-east), "usws" (us-west), "usct" (us-central), "emea" (europe middle east africa), "sg" (singapore)
+# Please deploy this config in each of your datacenters with the appropriate regional subdomain
+endpoint: "https://{{.Host}}.ads.lemmatechnologies.com/lemma/servad?src=prebid&pid={{.PublisherID}}&aid={{.AdUnit}}"
+geoscope:
+  - global
+# regional subdomain to be passed in the extra_info field as the "host". 
+# dooh-specific regional subdomain can also be optionally provided as "dooh-host". Available options are: "doohus" (US region), "doohsg" (Asia region)
+extra_info: "{\"host\":\"sg\"}"
```

Comment:
Let me make sure I'm following the DOOH scenario. Can the `extra_info` be structured like this:

```
"{\"host\":\"sg\", \"dooh-host\":\"doohsg\"}
```
And your adapter will then choose 'dooh-host' for DOOH requests and 'host' for non-DOOH? That would work.

Reply:
@bretg , 
Yes, the `extra_info` can also be structured like you've mentioned. We've kept the `dooh-host` as optional so even if it is not set, the bidder will choose the appropriate dooh specific host based on the chosen `host` parameter.